gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/OrganizationUserServiceImpl.java

```diff
@@ -191,4 +189,11 @@ public Single<User> updateLogoutDate(ReferenceType referenceType, String referen
                     return getUserService().update(user);
                 });
     }
+
+
+    @Override
+    public Single<AccountAccessToken> createAccountAccessToken(String organizationId, String userId, NewAccountAccessToken newAccountToken, io.gravitee.am.identityprovider.api.User principal) {
+        return getUserService().findById(ReferenceType.ORGANIZATION, organizationId, userId)
+                .flatMap(user -> getUserService().generateAccountAccessToken(user, newAccountToken));
```

Comment:
**todo**: it could be nice to generate an audit for this action in order to trace the fact that a token has been generated. (with success or not) You have an example here https://github.com/gravitee-io/gravitee-access-management/blob/master/gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/UserServiceImpl.java#L228.